vendor/composer/autoload_real.php

```diff
@@ -22,6 +22,8 @@ public static function getLoader()
             return self::$loader;
         }
 
+        require __DIR__ . '/platform_check.php';
```

Comment:
I think you overlooked the `require __DIR__ . '/platform_check.php';`.
I remember that you intentionally removed that before!
So we should remove this here as well as the whole `platform_check.php` file